applications/launchpad/backend/src/main.rs

```diff
@@ -164,3 +170,46 @@ fn handle_cli_options(cli_config: &CliConfig, pkg_info: &PackageInfo) {
         },
     }
 }
+
+async fn check_identity() {
```

Comment:
these look like debug-kinda functions and should be removed for the PR to be ready to review